tests/sentry/utils/performance_issues/test_consecutive_http_detector.py

```diff
@@ -137,3 +137,28 @@ def test_does_not_detect_with_high_duration_between_spans(self):
         ]  # ensure spans don't overlap
 
         assert self.find_problems(create_event(spans)) == []
+
+    def test_fingerprints_match_with_duplicate_http(self):
+        span_duration = 2000
+        spans = [
+            create_span("http.client", span_duration, "GET /api/endpoint1", "hash1"),
+            create_span("http.client", span_duration, "GET /api/endpoint2", "hash2"),
+            create_span("http.client", span_duration, "GET /api/endpoint3", "hash3"),
+        ]
+
+        spans = [
+            modify_span_start(span, span_duration * spans.index(span)) for span in spans
+        ]  # ensure spans don't overlap
+
+        problem_1 = self.find_problems(create_event(spans))[0]
+
+        spans.append(
+            modify_span_start(
+                create_span("http.client", span_duration, "GET /api/endpoint3", "hash3"), 6000
+            )
+        )
+
+        problem_2 = self.find_problems(create_event(spans))[0]
+
+        assert problem_2.fingerprint == "1-1009-30ce2c8eaf7cae732346206dcd23c3f016e75f64"
```

Comment:
what did this fingerprint look like before? was it `1-1009-30ce2...-30ce2...-30ce2...` ?

Reply:
It was a different value, `1-1009-169fcfe4c6ec972ca93f86df5d7a4a90418bea5e`.

I thought it was a good idea to assert the fingerprint on a hardcoded string, because if we make a change that affects the fingerprint, i would want tests to fail in order to know where the fingerprint is different.